scripts/istat/geos/preprocess.py

```diff
@@ -0,0 +1,96 @@
+# Lint as python3
+"""Preprocess the ISTAT geo code to NUTS code for importing into Data Commons"""
+
+import pandas as pd
+
+FILE_PATH_it = "./Elenco-codici-statistici-e-denominazioni-al-01_01_2020_it.csv"
```

Comment:
Can the download of these files be part of the script?

Reply:
Done